### PR TITLE
fix: swipe refresh success message display

### LINKED ISSUE #437: Shows refresh complete message even when it fails
Currently `Refresh complete` message is shown on swipe refresh completion. which misleads failed requests too. 

Expected:
The success message should be shown only on successful request. otherwise the resp error message should be shown in the snackbar.

### PR DESCRIPTION
**Pull request by mejariamol**
- added boolean parameter success to onRefreshComplete method of refreshable view
- modified refreshable observable transformer to distinguish in failure and success

fixes #437

### CODE REVIEW

**Comment by mejariamol** [comment]
@iamareebjamal please review.

File: app/src/main/java/org/fossasia/openevent/app/module/attendee/list/AttendeesFragment.java [hunk 1]
```diff
@@ -244,9 +242,10 @@ public void showProgress(boolean show) {
     }
 
     @Override
-    public void onRefreshComplete() {
+    public void onRefreshComplete(boolean success) {
         refreshLayout.setRefreshing(false);
-        Snackbar.make(binding.rvAttendeeList, R.string.refresh_complete, Snackbar.LENGTH_SHORT).show();
+        if (success)
+            ViewUtils.showSnackbar(binding.rvAttendeeList, R.string.refresh_complete);
```
Comment (mejariamol) [inline app/src/main/java/org/fossasia/openevent/app/module/attendee/list/AttendeesFragment.java]: on failure, we are already showing a message using ErrorUtils.

Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/openevent/app/module/attendee/list/AttendeesFragment.java]: OK

Comment (mejariamol) [inline app/src/main/java/org/fossasia/openevent/app/module/attendee/list/AttendeesFragment.java]: @iamareebjamal please review.

Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/openevent/app/module/attendee/list/AttendeesFragment.java]: Let the build fix first :smile: It's approved

File: app/src/main/java/org/fossasia/openevent/app/common/app/rx/ViewTransformers.java [hunk 2]
```diff
@@ -60,9 +60,13 @@ public static CompletableTransformer disposeCompletable(CompositeDisposable comp
...
         return observable ->
-            observable.doFinally(() -> {
-                if (forceReload) view.onRefreshComplete();
-            });
+            observable
+                .doOnComplete(() -> {
+                    if (forceReload) view.onRefreshComplete(true);
+                })
+                .doFinally(() -> {
+                    if (forceReload) view.onRefreshComplete(false);
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/openevent/app/common/app/rx/ViewTransformers.java]: This should be doOnError. Just noticed

Comment (mejariamol) [inline app/src/main/java/org/fossasia/openevent/app/common/app/rx/ViewTransformers.java]: Correct. Thanks

**Comment by mejariamol** [comment]
@iamareebjamal please review.

**Pull request merged**